WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormViewController.swift

```diff
@@ -64,12 +64,15 @@ final class ProductFormViewController<ViewModel: ProductFormViewModelProtocol>:
     private var newVariationsPriceSubscription: AnyCancellable?
     private var productImageStatusesSubscription: AnyCancellable?
 
+    private let showGroupedTableViewAppearance: Bool
```

Comment:
New design looks more like `grouped`  tableview, compared to previous `plain` variant. Since we can't change `UITableView` `style` in the runtime (to switch it based on the feature flag) - we have to fake appearance with custom headers and dividers.

Another cleaner solution could be to remove the `xib` file and initialize VC and `UITableView` programmatically. Then we can switch style in `init` based on some conditions.
This could be done after the feature is released to 100% users - remove `xib` file, switch VC to code init, hardcode grouped style permanently, remove headers/dividers hack.

Reply:
Thanks for the added explanation about these changes. I've opened #8928 to keep track of this cleanup once we've decided which experience to keep.